charts/rekor/values.yaml

```diff
@@ -26,8 +26,8 @@ redis:
     registry: docker.io
     repository: redis
     pullPolicy: IfNotPresent
-    # -- 6.2.6-alpine3.15
-    version: "sha256:6c42cce2871e8dc5fb3e843ed5c4e7939d312faf5e53ff0ff4ca955a7e0b2b39"
+    # -- 7.0.4-alpine3.16
+    version: sha256:dc1b954f5a1db78e31b8870966294d2f93fa8a7fba5c1337a1ce4ec55f311bc3
```

Comment:
i haven't updated these images before, but it might make sense to update each image version separately, just in case the release fails we know what went wrong 🤔 

@cpanato what do you usually do here?

Reply:
yes, can make incremental, it is just a bit more work, only if that is good to be bumped, with i dont know since that is a major bump from 6 to 7 maybe there is a breaking change

Reply:
Only bumped to newer patch version now.